src/test/java/uk/gov/cshr/civilservant/service/OrganisationalUnitServiceBHTest.java

```diff
@@ -311,19 +311,5 @@ public void givenAnOrgWithThreeLevelsAndThirdLevelIsRequested_whenGetOrganisatio
         assertThat(actual).extracting((OrganisationalUnit::getName)).containsOnly(
                 "child 0 of the god1");
     }
```

Comment:
Please can you just delete this file.  (I have deleted it in the past a few times, however it seems to reappear).